File: data/code_pairs/fortran/1676.f90
! Main program to test the SECOND function
PROGRAM testSecond
  IMPLICIT NONE
  REAL :: result

  result = SECOND()
  IF (result == 0.0E+0) THEN
    PRINT *, 'Test Passed!'
  ELSE
    PRINT *, 'Test Failed!'
  END IF
END PROGRAM testSecond

! Function SECOND returns a constant value
REAL FUNCTION SECOND()
    SECOND = 0.0E+0
END FUNCTION SECOND

File: data/code_pairs/cpp/1676.cpp
#include <iostream>
#include <cmath> // For std::abs

// Function to be tested
double SECOND() {
    return 0.0;
}

// Test function
void test_SECOND() {
    double result = SECOND();
    const double epsilon = 1e-7; // Tolerance for floating-point comparison

    if (std::abs(result - 0.0) < epsilon) {
        std::cout << "Test Passed!" << std::endl;
    } else {
        std::cout << "Test Failed! Result was " << result << std::endl;
    }
}

// Main function to run the test
int main() {
    test_SECOND();
    return 0;
}